### PR TITLE
Feature: Rc4 Account Setting fixes

### PR DESCRIPTION
**Pull request by adrianbparra**
Added and updated Settings Page.

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import thunk from 'redux-thunk';
 import configureMockStore from 'redux-mock-store';
-import fetchMock from 'fetch-mock';
+// import fetchMock from 'fetch-mock';
```
Comment (PrinceD96) [inline src/App.test.js:5]: Commented out code

File: src/App.test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import Enzyme, { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 // import expect from 'jest'
```
Comment (PrinceD96) [inline src/App.test.js:12]: Commented out code

File: src/components/settings/Settings.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        </div>
+      </div>
+    );
+  }
+
+const mapStateToProps = state => {
+	return {
+		isFetching: state.settings.loggedInUser.isFetching,
+		error: state.settings.loggedInUser.error,
+		// loggedUserInfo: state.loggedUserInfo,
```
Comment (PrinceD96) [inline src/components/settings/Settings.js:152]: Commented out code

File: src/components/settings/editaccount.js [hunk 4]
```diff
@@ -1,168 +1,240 @@
...
+    lastName: yup.string().required('Last Name Required'),
+    city: yup.string(),
+    state: yup.string(),
+});
 
 
 function EditAccount(props){
     /*
-        User can enter updates to first name, last name, email, or password.
+        User can enter updates to first name, last name, or password.
```
Comment (PrinceD96) [inline src/components/settings/editaccount.js:62]: Ok this one is a good one lol

File: src/App.test.js (hunk 2 above)
Comment (adrianbparra) [reply src/App.test.js:12]: I am going to leave the testing as is.

File: src/components/settings/editaccount.js (hunk 4 above)
Comment (adrianbparra) [reply src/components/settings/editaccount.js:62]: Lol well I forgot to remove it.

**Pull request merged**